pkgs/development/python-modules/seahub/default.nix

```diff
@@ -0,0 +1,56 @@
+{ stdenv, lib, fetchFromGitHub, python3Packages, makeWrapper }:
+
+python3Packages.buildPythonPackage rec {
+  pname = "seahub";
+  version = "8.0.7";
+
+  src = fetchFromGitHub {
+    owner = "haiwen";
+    repo = "seahub";
+    rev = "4f7bb3f617dd847cf0a6b33c0bfb567b44c06059";
```

Comment:
```suggestion
    rev = "v${version}-server";
```

Reply:
Unfortunately, upstream has the tendency to sometimes re-tag new revisions, causing the hash to change.
Yes, bad behaviour, but thus it makes sense to keep revisions as explicit hashes.

Reply:
Then please add a comment explaining the situation for the next person looking at this,